svthalia/concrexit
# Thabloid proxy user

Part of #2630 


### Summary
A clear and concise description of the changes that you made. What bug did you solve? Or what feature did you add?

I added the models proxy model ThabloidUser and the BlacklistedThabloidUser in the thabloid app.  Furthermore, I created an Admin for displaying all the ThabloidUsers and showing whether they want to receive it or not. The user can change the value of the field when editing his profile. 

### How to test
Steps to test the changes you made:
1. Go to the ThabloidUser admin to see the current status of the field
2. Edit the field of the user on the edit_profile page
3. Go back to the ThabloidUser to see whether the field was actually changed.
